### PR TITLE
Using Nan 2.0

### PR DESCRIPTION
**Pull request by 5at**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,27 @@
 language: node_js
+
+env:
+  - CXX=g++-4.8
```
Comment (parkr) [inline .travis.yml]: What features of g++ 4.8 are required? Is gcc llvm supported?

**Comment by parkr** [comment]
Just one concern about supporting earlier versions of GCC. @FooBarWidget, any objections?

**Comment by 5at** [comment]
no I havent. i just copied what others are doing to upgrade to Nan 2.0.  Maybe Nan 2.0 requires it? only one way to find out :)

**Comment by 5at** [comment]
Just to clarify:
Its not that any code in node-sha3 needs g++ v4.8 features, it's that the compiler chosen for travis' iojs-3 setup does not compile and the errors are in the v8 headers (???).
When i removed all the lines about what version of g++ to use ALL-BUT-ONE of the node-js setups compiled fine in travis.
I'm pretty sure people can still use whatever compiler is specified for the node version they are using. (Unless Nan 2.0 has specified a minimum g++ version ??)
Again this is not saying that ALL usage of this module has to use, at least, version 4.8 of g++.  It is only for passing travis' current setup.

**Comment by parkr** [comment]
Thanks for the clarification, @5at. LGTM.

**Pull request merged**